Plans/Sentry_Base.xctestplan

```diff
@@ -33,7 +33,64 @@
         "SentryCrashReportStore_Tests\/testCrashReportCount1",
         "SentryCrashReportStore_Tests\/testDeleteAllReports",
         "SentryCrashReportStore_Tests\/testStoresLoadsMultipleReports",
+        "SentryHttpTransportTests",
```

Comment:
I think instead of doing this we should also move them into a new unit test target. This will not age well.

Reply:
I am not entirely happy with this aswell. My idea was not having to manage multiple targets and making sure the settings match (even though xcodegen makes this easier already) and instead just run specific test suites separately.